README.md

```diff
@@ -14,87 +14,87 @@ Supports Node.js v4, v6 and v7 on Linux, MacOS, Windows and AIX.
+node-report.triggerReport();
 ```
 The API can be used without adding the automatic exception and fatal error
 hooks and the signal handler, as follows:
 
 ```js
-var nodereport = require('nodereport/api');
-nodereport.triggerReport();
+var node-report = require('node-report/api');
+node-report.triggerReport();
 ```
 
-Content of the NodeReport consists of a header section containing the event
+Content of the node-report consists of a header section containing the event
 type, date, time, PID and Node version, sections containing JavaScript and
 native stack traces, a section containing V8 heap information, a section
 containing libuv handle information and an OS platform information section
-showing CPU and memory usage and system limits. An example NodeReport can be
+showing CPU and memory usage and system limits. An example node-report can be
 triggered using the Node.js REPL:
 
 ```
 $ node
-> nodereport = require('nodereport')
-> nodereport.triggerReport()
-Writing Node.js report to file: NodeReport.20161020.091102.8480.001.txt
+> node-report = require('node-report')
+> node-report.triggerReport()
+Writing Node.js report to file: node-report.20161020.091102.8480.001.txt
 Node.js report completed
 >
 ```
 
-When a NodeReport is triggered, start and end messages are issued to stderr
+When a node-report is triggered, start and end messages are issued to stderr
 and the filename of the report is returned to the caller. The default filename
 includes the date, time, PID and a sequence number. Alternatively, a filename
 can be specified as a parameter on the `triggerReport()` call.
 
 ```js
-nodereport.triggerReport("myReportName");
+node-report.triggerReport("myReportName");
 ```
 
 ## Configuration
 
 Additional configuration is available using the following APIs:
 
 ```js
-nodereport.setEvents("exception+fatalerror+signal+apicall");
-nodereport.setSignal("SIGUSR2|SIGQUIT");
-nodereport.setFileName("stdout|stderr|<filename>");
-nodereport.setDirectory("<full path>");
-nodereport.setCoreDump("yes|no");
-nodereport.setVerbose("yes|no");
+node-report.setEvents("exception+fatalerror+signal+apicall");
+node-report.setSignal("SIGUSR2|SIGQUIT");
+node-report.setFileName("stdout|stderr|<filename>");
+node-report.setDirectory("<full path>");
+node-report.setCoreDump("yes|no");
+node-report.setVerbose("yes|no");
 ```
 
 Configuration on module Initialization is also available via environment variables:
 
 ```bash
-export NODEREPORT_EVENTS=exception+fatalerror+signal+apicall
-export NODEREPORT_SIGNAL=SIGUSR2|SIGQUIT
-export NODEREPORT_FILENAME=stdout|stderr|<filename>
-export NODEREPORT_DIRECTORY=<full path>
-export NODEREPORT_COREDUMP=yes|no
-export NODEREPORT_VERBOSE=yes|no
+export node-report_EVENTS=exception+fatalerror+signal+apicall
+export node-report_SIGNAL=SIGUSR2|SIGQUIT
+export node-report_FILENAME=stdout|stderr|<filename>
+export node-report_DIRECTORY=<full path>
+export node-report_COREDUMP=yes|no
+export node-report_VERBOSE=yes|no
```

Comment:
We want to keep the environment variables case (upper), probably without the hyphen.